java/src/main/java/com/saucelabs/simplesauce/Options.java

```diff
@@ -0,0 +1,274 @@
+    }
+
+    private static final class PreRunLookup {
+        private static final Map<String, String> lookup = new HashMap<String, String>();
+    }
+
+    public enum PageLoadStrategy {
+        NONE("none"),
+        EAGER("eager"),
+        NORMAL("normal");
+
+        @Getter private String value;
+
+        public static Set keys() {
+            return PageLoadStrategyLookup.lookup.keySet();
+        }
+
+        PageLoadStrategy(String value) {
+            this.value = value;
+            PageLoadStrategyLookup.lookup.put(value.toString(), this.name());
+        }
+
+        public static String fromString(String value) {
+            return PageLoadStrategyLookup.lookup.get(value);
+        }
+
+        public String toString() {
+            return this.value.toString();
+        }
+    }
+
+    public enum UnhandledPromptBehavior {
+        DISMISS("dismiss"),
+        ACCEPT("accept"),
+        DISMISS_AND_NOTIFY("dismiss and notify"),
+        ACCEPT_AND_NOTIFY("accept and notify"),
+        IGNORE("ignore");
+
+        @Getter private String value;
+
+        public static Set keys() {
+            return UnhandledPromptBehaviorLookup.lookup.keySet();
+        }
+
+        UnhandledPromptBehavior(String value) {
+            this.value = value;
+            UnhandledPromptBehaviorLookup.lookup.put(value.toString(), this.name());
+        }
+
+        public static String fromString(String value) {
+            return UnhandledPromptBehaviorLookup.lookup.get(value);
+        }
+
+        public String toString() {
+            return this.value.toString();
+        }
+    }
+
+    public enum Prerun {
+        EXECUTABLE("executable"),
+        ARGS("args"),
+        BACKGROUND("background"),
+        TIMEOUT("timeout");
+
+        @Getter private String value;
+
+        public static Set keys() {
+            return PreRunLookup.lookup.keySet();
+        }
+
+        Prerun(String value) {
+            this.value = value;
+            PreRunLookup.lookup.put(value, this.name());
+        }
+
+        public static String fromString(String value) {
+            return PreRunLookup.lookup.get(value);
+        }
+
+        public String toString() {
+            return this.value.toString();
+        }
+    }
+
+    public enum JobVisibility {
+        PUBLIC("public"),
+        PUBLIC_RESTRICTED("public restricted"),
+        SHARE("share"),
+        TEAM("team"),
+        PRIVATE("private");
+
+        @Getter private String value;
+
+        public static Set keys() {
+            return JobVisibilityLookup.lookup.keySet();
+        }
+
+        JobVisibility(String value) {
+            this.value = value;
+            JobVisibilityLookup.lookup.put(value.toString(), this.name());
+        }
+
+        public static String fromString(String value) {
+            return JobVisibilityLookup.lookup.get(value);
+        }
+
+        public String toString() {
+            return this.value.toString();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid calling toString() on String objects; this is unnecessary.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4758011)